fundingmanager.go

```diff
@@ -1283,8 +1338,12 @@ func (f *fundingManager) waitForFundingWithTimeout(completeChan *channeldb.OpenC
 			// The fundingManager is shutting down, will resume
 			// waiting for the funding transaction on startup.
 			return
-		case <-waitingDoneChan:
-			close(doneChan)
+		case shortChanID, ok := <-waitingConfChan:
+			if !ok {
+				close(confChan)
```

Comment:
Add comment
// Failed waiting for confirmation, close confChan to indicate failure.